src/LinkBlockPresenter.php

```diff
@@ -119,11 +119,30 @@ private function makeCustomLinks($customContent)
                     'title' => $el['title'],
                     'description' => '',
                     'url' => $el['url'],
+                    'target' => $this->isExternalLink($el['url']),
```

Comment:
Since the property is called "target", for flexibility's sake I'd prefer its value to be "_blank" or "" instead of true/false.